File: INICIANTE/1044/main.c
#include <stdbool.h>
#include <stdio.h>

bool isMultiple(int a, int b) {
    return a % b == 0 || b % a == 0;
}

int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    printf(isMultiple(a, b) ? "Sao Multiplos\n" : "Nao sao Multiplos\n");
    return 0;
}

File: INICIANTE/1044/main.js
function isMultiple(a, b) {
    return a % b === 0 || b % a === 0;
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let [a, b] = lines[0].split(' ').map(Number);
console.log(isMultiple(a, b) ? 'Sao Multiplos' : 'Nao sao Multiplos');
